packages/devtools_app/integration_test/run_tests.dart

```diff
@@ -57,6 +75,14 @@ void main(List<String> testRunnerArgs) async {
   }
 }
 
+bool _isUnsupportedForDevice(String testPath, {required String testAppDevice}) {
+  if (testAppDevice == 'flutter-tester') return false;
```

Comment:
can we create an enum for the testAppDevice types? `flutter-tester` and `chrome` for now, but likely `cli` in the future as well.

Reply:
Done!